src/frontend/App/hooks/useHentOppgaverForOpprettelse.ts

```diff
@@ -0,0 +1,61 @@
+import { RessursFeilet, RessursStatus, RessursSuksess } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+import { useCallback, useState } from 'react';
+import {
+    IOppgaverForOpprettelse,
+    OppgaveForOpprettelseType,
+} from '../../Komponenter/Behandling/Totrinnskontroll/OppgaverForOpprettelse';
+
+export interface OppgaverForOpprettelseState {
+    feilmelding: string | undefined;
+    hentOppgaverForOpprettelse: (behandlingId: string) => void;
+    oppgaverForOpprettelse: IOppgaverForOpprettelse;
+    oppdaterOppgavetyperSomSkalOpprettes: (typer: OppgaveForOpprettelseType[]) => void;
+}
+
+const initialState: IOppgaverForOpprettelse = {
+    oppgavetyperSomSkalOpprettes: [],
+    oppgavetyperSomKanOpprettes: [],
+};
```

Comment:
Hvis de to andre statene kan splittes vil initalstate for begge være [] og denne kan slettes